templates/documentation/delivery-analytics/private-video-delivery-with-mp4-built-in-players.md

```diff
@@ -1,7 +1,8 @@
 ---
-title: "Private video delivery with MP4 built-in players"
+title: Private mp4 video playback and delivery
+slug: private-video-delivery-with-mp4-built-in-players
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅